mbzirc_ign/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ find_package(ignition-fuel_tools7 REQUIRED)
 find_package(ignition-math6 REQUIRED)
 set(IGN_MATH_VER ${ignition-math6_VERSION_MAJOR})
 find_package(ignition-msgs8 REQUIRED)
+set(IGN_MSGS_VER ${ignition-msgs8_VERSION_MAJOR})
```

Comment:
Nit: Do we need this?

Reply:
removed. [de1da17](https://github.com/osrf/mbzirc/pull/52/commits/de1da1726a12615716efd5b1d684e417c7ce8ea8)